### ISSUE TITLE
KeyError: 'wasm'

### ISSUE THREAD

**Issue by rviscomi** [HTTPArchive/data-pipeline#243]
```
KeyError: 'wasm'

at .aggregate_stats ( /opt/apache/beam-venv/beam-venv-worker-sdk-0-0/lib/python3.8/site-packages/modules/transformation.py:577 )
at .generate_pages ( /opt/apache/beam-venv/beam-venv-worker-sdk-0-0/lib/python3.8/site-packages/modules/transformation.py:184 )
at .get_page ( /opt/apache/beam-venv/beam-venv-worker-sdk-0-0/lib/python3.8/site-packages/modules/import_all.py:67 )
```

https://console.cloud.google.com/errors/detail/CLTnrKT98r6TmQE;time=P30D?project=httparchive

**Comment by tunetheweb** [comment]
Oh this sounds familiar. I was touching that recently...

**Comment by tunetheweb** [comment]
In the oddly coincidental PR number #234 !

**Comment by tunetheweb** [comment]
Ah I never released that fix - see #236 . That explains why we're seeing it again. And think, like January, it should still be fine. Not sure if we'll lose a few tests or if it's a non-critical error.
